__tests__/index.test.js

```diff
@@ -6,9 +6,7 @@ describe("Home", () => {
   it("renders an octopus", () => {
     render(<Home />);
 
-    const heading = screen.getByRole("heading", {
-      name: /🐙/i,
-    });
+    const heading = screen.getByRole("button");
 
     expect(heading).toBeInTheDocument();
```

Comment:
if we fake the test, let's fake with style: If you select the button, please call the variable `button` instead of `heading` too...